docs/usage/input.md

```diff
@@ -52,20 +52,23 @@ The following sections describe the columns that can be used in every sample-she
 ## Columns: CRAM
 | column                  | type     | required | default      | description                                                                                                 |
 |-------------------------|----------|----------|--------------|-------------------------------------------------------------------------------------------------------------|
+| ``bed``                 | ``file`` |          |              | Bed file to filter the input data with. allowed file extensions: [``bed``]                                  |

```

Comment:
what is the behavior for reads partially overlapping a region in the bed file?

Reply:
all reads overlapping the bed file.
something we need to think about, if this is expected/desired behaviour